src/Files.App/Views/MainPage.xaml

```diff
@@ -272,12 +271,7 @@
 						BorderBrush="{ThemeResource CardStrokeColorDefaultBrush}"
```

Comment:
I think the background and border can be removed from here.

Reply:
Done

Reply:
Thanks